lib/hooks/useTime.ts

```diff
@@ -0,0 +1,36 @@
+import { useEffect, useState } from 'react';
+import { formatter } from '../utils';
+
+const SunCalc = require('suncalc');
+
+/**
+ * Creates a hook more accessesing specific values like the current time, whether it is daytime
+ * @param latitude, the latitude float value that is passed in
+ * @param longitude, the longitude number value that is passed in
+ * @returns currentTimeNoFilter, unfiltered version of the current time (right now) that a user can format
+ * @returns isDaytime, boolean value indicating whether it is daytime or not based on current time zone.
+ */
+const useTime = (latitude: number, longitude: number) => {
+  const [currentTimeNoFilter, setCurrentTimeNoFilter] = useState(Date.now());
```

Comment:
don't think we need to specify `NoFilter` here—seems like we can just call these`[currentTime, setCurrentTime]`